source/MaterialXTest/ShaderValid.cpp

```diff
@@ -464,7 +486,9 @@ static void runOSLValidation(const std::string& shaderName, mx::TypedElementPtr
         bool validated = false;
         try
         {
-            validator.setOslOutputFilePath(shaderPath);
+            // Set output path and shader name
```

Comment:
Cleaned up this so the output file path doesn't contain the shader name. Need to use both the output path + the shader name in different places so separating it out.